packages/common/src/api/tan-query/comments/useComments.ts

```diff
@@ -53,11 +55,14 @@ export const useComments = (
     return byId
   }, [comments])
 
-  return {
-    data: comments,
-    byId,
+  const results = {
     ...queryResults,
     isPending,
     isLoading
+  } as typeof queryResults
```

Comment:
huh slightly annoying we have to cast it, but fine

Reply:
yeah it is a bit weird, i can relax the status combiner props if we want? in a way i think this is good cause it ensures our hooks are returning the right expected types so they truly act like tan-query? ie when isPending is true, data will be undefined

Reply:
yeah totally. hooks should return a proper QueryResults type